packages/components/src/Tabs/Tabs.test.tsx

```diff
@@ -110,28 +128,34 @@ test('cursor indicates interactivity', () => {
   );
   const tabs = screen.getAllByRole('tab');
   expect(tabs[0].className).toMatchInlineSnapshot(
-    `"flex cursor-pointer justify-center aria-disabled:cursor-not-allowed min-h-[40px] data-[hover]:text-tabs-tab-text data-[hover]:border-b-tabs-tab-hover data-[hover]:border-b-8 data-[hover]:border-solid disabled:text-tabs-tab-disabled selected:border-b-primary-600 selected:border-b-8 selected:border-solid px-2 pb-2 text-lg"`
+    `"flex cursor-pointer justify-center aria-disabled:cursor-not-allowed selected:border-red-500 selected:border-b-8 selected:border-solid px-2 pb-2 text-lg"`
   );
   expect(tabs[1].className).toMatchInlineSnapshot(
-    `"flex cursor-pointer justify-center aria-disabled:cursor-not-allowed min-h-[40px] data-[hover]:text-tabs-tab-text data-[hover]:border-b-tabs-tab-hover data-[hover]:border-b-8 data-[hover]:border-solid disabled:text-tabs-tab-disabled selected:border-b-primary-600 selected:border-b-8 selected:border-solid px-2 pb-2 text-lg"`
+    `"flex cursor-pointer justify-center aria-disabled:cursor-not-allowed selected:border-red-500 selected:border-b-8 selected:border-solid px-2 pb-2 text-lg"`
   );
 });
 
-test('open tabpanel when its tab controller is clicked', () => {
+test('allow styling TabPanel & container via theme', () => {
   render(
-    <Tabs>
-      <Tabs.Item title="tab1" key="1">
+    <Tabs
+      aria-label="tabs container"
+      className="flex gap-2"
+      disabledKeys={['2']}
+    >
+      <Tabs.Item className="bg-green-300" key="1" title={'tab1'}>
         tab-1 content
       </Tabs.Item>
-      <Tabs.Item title="tab2" key="2">
+      <Tabs.Item key="2" title={'tab2'}>
         tab-2 content
```

Comment:
can I also set a variant on the `<Tab.Panel>` ?

Reply:
no we can't , why should we pass variant for `<Tabs.Item>` ?

Reply:
> no we can't , why should we pass variant for `<Tabs.Item>` ?

@sarahgm

Reply:
don't know maybe we need this in future, or how can I set styles on the panel?

Reply:
you can style panel like this : 
`
<Tabs.Item className='bg-red-300'>
   tab panel content to be styled 
</Tabs.Item>
`
so the `className` here is for tab panel not for the tab button 
@sarahgm

Reply:
but how would a user than style it?

Reply:
you mean the tab button ?!  
if that what u mean then idk 
because className is for tab panel but what if you want to style the tab button ?! then you should make this className reserved for tab  button instead of tab panel 
what i mean here you only have one className not two then you either chose to style for tab panel or button ?

Reply:
Ehhhh Now I'm also confused :D :D @sebald you as author of the issue can you explain to me? :D

Reply:
We want the variant to be used by the container, tab, tablist and tabpanel :) Does this help?